ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -337,14 +337,9 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   }
 
   // At this point, we have a response from the backend for all queries. (We're past loading === true).
-  // If we don't data from any of the queries, then check if we want to show an error.
-  // Put another way: If any queries have data, even if other queries failed, we will show the data (not the error).
-  // Unfortunately, partial errors get swallowed in this case.
-  // This could be refactored when someone takes a look at validation and error-handling.
-  if (!hasData) {
-    for (const result of queryResults) {
-      if (result.error) throw result.error;
-    }
+  // Then check if we want to show an error, no matter if we have data or not.
```

Comment:
This one is to be checked further. I had to do this because when switching from table to graph with a query valid for table but not for graph, the result was not showing the error. There was data and also error. 

Maybe a better investigation would be: why there was data if the result was an error, maybe a merge result that tried to merge from previous.

Reply:
I found out my problem I was not putting mode in the query key of the use queries. After adding it I can get back this code as it was.